marklogic-data-hub-spark-connector/src/main/java/com/marklogic/hub/spark/sql/sources/v2/DefaultSource.java

```diff
@@ -75,17 +64,28 @@ public static HubClientConfig buildHubClientConfig(Map<String, String> options)
         hubClientConfig.applyProperties(props);
         return hubClientConfig;
     }
+
+    @Override
```

Comment:
Do these need to move as part of this PR? I'm planning on moving buildHubClientConfig out of here soon - the Reader will need it. In which case these two methods can stay put.

Reply:
The changes are just formatting in this PR.